prusti-interface/src/specs/decoder.rs

```diff
@@ -25,8 +25,16 @@ impl<'a, 'tcx> DefSpecsDecoder<'a, 'tcx> {
         }
     }
 
-    // From rustc
     fn def_path_hash_to_def_id(&self, hash: DefPathHash) -> DefId {
+        // Sanity check
+        let cstore = std::panic::AssertUnwindSafe(self.tcx.cstore_untracked());
+        let result = std::panic::catch_unwind(|| {
+            cstore.stable_crate_id_to_crate_num(hash.stable_crate_id())
+        });
+        if result.is_err() {
+            panic!("A compiled dependency is out of sync. Try deleting the target folder, or that of `prusti-contracts`.")
```

Comment:
Regarding "target folder ... of `prusti-contracts`", isn't this PR removing that folder? Or is the message for those that still have that folder around from before this PR?

Reply:
Ooops yeah I forgot to change the message